.github/workflows/workflow.yml

```diff
@@ -37,9 +38,9 @@ jobs:
         name: cate-jupyter-build-push-docker-image-latest
         with:
           image: ${{ env.ORG_NAME }}/${{ env.APP_NAME }}
-          tags: ${{ env.CATE_VERSION }}
+          tags: v3.0.1.dev1_mooc_nbs
```

Comment:
I do not understand this tag - how does it have an effect then on the k8s-configs for cate-jh?

Reply:
Yes, I stumbled across this after the PR. I have reverted it so that the cate version is used again.